.github/workflows/slash-command-dispatch.yml

```diff
@@ -0,0 +1,19 @@
+name: Slash Command Dispatch
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  slashCommandDispatch:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Slash Command Dispatch
+        uses: peter-evans/slash-command-dispatch@v2
+        with:
+          issue-type: pull-request
+          permission: write
+          # This token uses the public_repo scope to create repository_dispatch and workflow_dispatch events.
+          # This particular personal access token is tied to this account: https://github.com/aprea
+          # Note: GITHUB_TOKEN does not work here.
+          token: ${{ secrets.SLASH_COMMAND_ACCESS_TOKEN }}
```

Comment:
This requires a personal access token, for the time being, I'm using a token I generated on my own account. I don't suppose we have an Automattic/Woo CI type account that we could use instead?

Reply:
Good thinking! I couldn't find a general or woo/transact bot account searching Automattic org for `bot` or `auto`[mation], though there are some bot accounts for specific purposes.

I suggest P2ing this question internally to get alignment on how best to set this up. Maybe there's an existing account we can use, or may be we need a new one. Alternatively maybe there's an alternative mechanism to grant access (e.g. deploy key) (not sure if this is supported by the actions you're using).

The main consideration here is ensuring we don't grant more access than the automation needs ([principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege)).

Reply:
I'm assuming the permission / token is required for the emoji reactions and comment w/ build link

Reply:
> I don't suppose we have an Automattic/Woo CI type account that we could use instead?

`matticbot` user might be relevant, thanks @brucealdridge for [spotting this in Jetpack](https://github.com/Automattic/jetpack/blob/master/.github/files/gh-autotagger/workflows/autotagger.yml#L32-L35).

Reply:
Thanks all, I have switched to using a PAT generated on the @matticbot account, relevant request here pMz3w-eoq-p2